### ISSUE TITLE
Missing:  Geography data type and any example how to use

### ISSUE THREAD

**Issue by SheldonHage** [dotnet/EntityFramework.Docs#4222]
After many attempts at finding references to Geography (as opposed to Geometry) examples I have found that aspect of the EF Core Spatial support elusive.   If it exists, please update this document to include some examples.  I suggest showing how a LINQ query against a SQL table with a Geography column could return results within a Distance of some number of Meters from an origination Point.

Thank you

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c2dcf82b-f177-e91b-96a5-67f2f57bd6c4
* Version Independent ID: 89c0803f-570f-07a6-6dc2-285678712c86
* Content: [Spatial Data - EF Core](https://learn.microsoft.com/en-us/ef/core/modeling/spatial?source=recommendations)
* Content Source: [entity-framework/core/modeling/spatial.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/modeling/spatial.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @bricelam
* Microsoft Alias: **avickers**

**Comment by ajcvickers** [comment]
I agree this could be clearer. There is no "geography" type. Instead, everything is `Geometry` but with an SRID. For example, from that doc:

```C#
var seattle = new Point(-122.333056, 47.609722) { SRID = 4326 };
var redmond = new Point(-122.123889, 47.669444) { SRID = 4326 };

// In order to get the distance in meters, we need to project to an appropriate
// coordinate system. In this case, we're using SRID 2855 since it covers the
// geographic area of our data
var distanceInDegrees = seattle.Distance(redmond);
var distanceInMeters = seattle.ProjectTo(2855).Distance(redmond.ProjectTo(2855));
```
